farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.html

```diff
@@ -517,35 +549,57 @@ <h1 class="text-center">Seeding Report</h1>
                     let filteredTableData = tableData.filter((x) => x.data[3] === 'Direct')
                     let sumHours = 0
                     filteredTableData.map(h => {
-                        sumHours = sumHours + h.data[10]//*h.data[11])
+                        sumHours = sumHours + (h.data[10]*parseFloat(h.data[11]))
```

Comment:
I don't know why this was commented out originally, but it shouldn't be commented out.